packages/WaffleGrid/WaffleGrid.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Box from '@tds/core-box'
+import Link from '@tds/core-link'
+import Image from '@tds/core-image'
+import Paragraph from '@tds/core-paragraph'
+import { colorGreyGainsboro } from '@tds/core-colours'
+
+import { safeRest } from '@tds/util-helpers'
+
+const StyledLogoRow = styled(Box)({
+  flexWrap: 'wrap',
+  justifyContent: 'flex-start',
+  '@media(min-width:992px)': {
+    'div:not(:last-child)': {
+      borderRight: `1px solid ${colorGreyGainsboro}`,
+    },
+    'div:nth-of-type(6n)': {
+      borderRight: '0px',
+    },
+  },
+  '@media(min-width:576px) and (max-width:992px)': {
+    '> div:nth-of-type(1):nth-of-last-child(3)': {
+      borderBottom: '5px solid black',
+    },
+    'div:nth-of-type(3n - 1)': {
+      borderLeft: `1px solid ${colorGreyGainsboro}`,
+      borderRight: `1px solid ${colorGreyGainsboro}`,
+    },
+    'div:nth-of-type(3n - 1):last-child': {
+      borderLeft: `1px solid ${colorGreyGainsboro}`,
+    },
+    '> div:not(:last-child)': {
+      borderBottom: `1px solid ${colorGreyGainsboro}`,
+    },
+    'div:nth-last-of-type(-n+3):nth-of-type(3n) ~ *': {
+      borderBottom: `1px solid ${colorGreyGainsboro}`,
+      '&:last-child': {
+        borderRight: '0px',
+      },
+    },
+    'div:nth-last-of-type(-n+4):nth-of-type(3n) ~ *': {
+      borderBottom: '0px',
+    },
+  },
+  '@media(max-width:576px)': {
+    'div:nth-of-type(even)': {
+      borderLeft: `1px solid ${colorGreyGainsboro}`,
+    },
+    '> div:not(:last-child)': {
+      borderBottom: `1px solid ${colorGreyGainsboro}`,
+    },
+    '> div:nth-last-of-type(-n+2):not(:nth-of-type(even))': {
+      borderBottom: '0px',
+    },
+  },
+})
+
+const StyledLogoWrapper = styled(Box)({
+  '@media(max-width:575px)': {
+    flex: '0 1 50%',
+    width: '50%',
+    alignItems: 'center',
+    paddingBottom: '16px',
+    '~ div:nth-of-type(n + 3)': {
+      paddingTop: '16px',
+    },
+    '~ div:nth-last-of-type(-n + 2)': {
+      paddingBottom: '0px',
+    },
+  },
+  '@media(min-width:576px) and (max-width:991px)': {
```

Comment:
Can you replace the media queries with values from `import { media } from '@tds/core-responsive'`